apps/src/templates/levelSummary/FreeResponseAIEvaluation.tsx

```diff
@@ -0,0 +1,128 @@
+import Button from '@code-dot-org/component-library/button';
+import React, {useEffect, useState} from 'react';
+
+import {getChatCompletionMessage} from '@cdo/apps/aiTutor/chatApi';
+import CollapsibleSection from '@cdo/apps/templates/CollapsibleSection';
+
+import SafeMarkdown from '../SafeMarkdown';
+
+import style from '@cdo/apps/levelbuilder/ai-iteration-tools/ai-tutor/ai-tutor-tester.module.scss';
+
+interface StudentResponse {
+  user_id: number;
+  text: string;
+  student_display_name: string;
+  aiEvaluation: string;
+}
+
+interface FreeResponseAIEvaluationProps {
+  responses: StudentResponse[];
+  levelInstructions: string;
+}
+const FreeResponseAIEvaluation: React.FunctionComponent<
+  FreeResponseAIEvaluationProps
+> = ({responses, levelInstructions}) => {
+  const [evaluationsPending, setEvaluationsPending] = useState<boolean>(false);
+  const [evaluations, setEvaluations] = useState<StudentResponse[]>([]);
+  const [evaluationCount, setEvaluationCount] = useState<number>(0);
+  const [aiSummary, setAiSummary] = useState<string>('');
+  const evaluationComplete = responses.length === evaluationCount;
+
+  useEffect(() => {
+    if (evaluationComplete) {
+      setEvaluationsPending(false);
+      summarizeStudentEvaluations(evaluations);
+    }
+  }, [evaluations, evaluationComplete]);
+
+  const basePrompt =
+    'You are a teaching assistant for a high school AP Computer Science class where the students are learniing JavaScript.';
+
+  const getAIEvaluations = async () => {
+    setEvaluationsPending(true);
+    const responsePromises = responses.map(async studentResponse => {
```

Comment:
For my own knowledge here, why is the `async` needed here?  It looks like `evaluateStudentResponse` already returns a promise.  So is there a promise wrapped inside a promise here?